projects/kit/components/toast/toast.style.less

```diff
@@ -0,0 +1,37 @@
+@import '@taiga-ui/core/styles/taiga-ui-local';
+
+[tuiToast] {
+    --t-radius: 1rem;
```

Comment:
I don't think we need those variables, we can just override those values on host. Variables are helpful when you have different sizes and do not want to increase specificity due to data-size attribute in selector.